main.py

```diff
@@ -139,7 +139,7 @@ def markers():
     if request.values.get('format') == 'csv':
         return Response(generate_csv(accidents), headers={
             "Content-Type": "text/csv",
-            "Content-Disposition": 'attachment; filename="data.csv"'
+            "Content-Disposition": 'attachment; filename="Anyway-accidents-from-'+kwargs['start_date'].strftime('%Y-%m-%d')+'-to-'+kwargs['end_date'].strftime('%Y-%m-%d')+'.csv"'
```

Comment:
It's not a big deal but in  this case I would
1. extract the format specifier `%Y-%m-%d` to a variable
2. use [`format`](https://pyformat.info/#simple) rather than concatenating strings

Also, you should use a `#` before the issue number in the commit message, so GitHub will know to automatically associate it (and also close it if you say "fixed" in the message).